tests/e2e-tests/config/fixture-loaders.js

```diff
@@ -0,0 +1,297 @@
+const createProducts = () => {
+	return WooCommerce.post( 'products/batch', {
+		create: [
+			{
+				name: 'Woo Single #1',
+				type: 'simple',
+				regular_price: '21.99',
+				virtual: true,
+				downloadable: true,
+				downloads: [
+					{
+						name: 'Woo Single',
+						file:
+							'http://demo.woothemes.com/woocommerce/wp-content/uploads/sites/56/2013/06/cd_4_angle.jpg',
+					},
+				],
+				images: [
+					{
+						src:
+							'http://demo.woothemes.com/woocommerce/wp-content/uploads/sites/56/2013/06/cd_4_angle.jpg',
+					},
+				],
+			},
+		],
+	} )
+		.then( ( products ) => {
+			createReviews( products.data.create[ 0 ].id );
+			return products.data.create.map( ( product ) => product.id );
+		} )
+		.catch( ( e ) => console.log( e.response.data.message ) );
+};
+// deleting products will also delete review.
+const deleteProducts = ( ids ) => {
+	WooCommerce.post( 'products/batch', {
+		delete: ids,
+	} ).catch( ( error ) => console.log( error ) );
+};
+const createReviews = ( id ) => {
+	return WooCommerce.post( 'products/reviews/batch', {
+		create: [
+			{
+				product_id: id,
+				review: 'Looks fine',
+				reviewer: 'John Doe',
+				reviewer_email: 'john.doe@example.com',
+				rating: 4,
+			},
+			{
+				product_id: id,
+				review: 'I love this album',
+				reviewer: 'John Doe',
+				reviewer_email: 'john.doe@example.com',
+				rating: 5,
+			},
+			{
+				product_id: id,
+				review: 'a fine review',
+				reviewer: "John Doe' niece",
+				reviewer_email: 'john.doe@example.com',
+				rating: 5,
+			},
+		],
+	} ).catch( ( e ) => console.log( e.response.data.message ) );
+};
+const enableCheque = () => {
+	return WooCommerce.post( 'payment_gateways/cheque', {
+		enabled: true,
+	} ).catch( ( error ) => console.log( error ) );
+};
+const enablePaypal = () => {
+	return WooCommerce.post( 'payment_gateways/paypal', {
+		enabled: true,
+	} ).catch( ( error ) => console.log( error ) );
+};
+
+const enablePaymentGateways = () => {
+	Promise.all( [ enableCheque(), enablePaypal() ] );
+};
+
+const createShippingMethods = () => {
+	const UKShipping = () =>
+		WooCommerce.post( 'shipping/zones', { name: 'UK' } )
+			.then( ( response ) => {
+				return response.data.id;
+			} )
+			.then( ( zoneId ) => {
+				WooCommerce.put( `shipping/zones/${ zoneId }/locations`, {
+					code: 'UK',
+				} );
+				return zoneId;
+			} )
+			.then( ( zoneId ) => {
+				WooCommerce.post( `shipping/zones/${ zoneId }/methods`, {
+					method_id: 'flat_rate',
+					settings: {
+						title: 'Normal Shipping',
+						cost: '20.00',
+					},
+				} );
+				return zoneId;
+			} )
+			.then( ( zoneId ) => {
+				WooCommerce.post( `shipping/zones/${ zoneId }/methods`, {
+					method_id: 'free_shipping',
+					settings: {
+						title: 'Free Shipping',
+						cost: '00.00',
+						requires: 'coupon',
+					},
+				} );
+				return zoneId;
+			} )
+			.catch( ( e ) => console.log( e.response.data.message ) );
+
+	return Promise.all( [ UKShipping() ] );
```

Comment:
you can only create one shipping zone at a time, we can be refactored so that the create part is dynamic, for now, you assign the create promise to a variable `UKShipping` or anything and pass it to promise.all.